internal/sirius/person_test.go

```diff
@@ -46,6 +46,13 @@ func TestPerson(t *testing.T) {
 							"firstname": dsl.String("John"),
 							"surname":   dsl.String("Doe"),
 							"dob":       dsl.Term("01/01/1970", `^\d{1,2}/\d{1,2}/\d{4}$`),
+							"children": dsl.Like([]map[string]interface{}{{
+								"id":        dsl.Like(105),
+								"uId":       dsl.Term("7000-0000-0002", `7\d{3}-\d{4}-\d{4}`),
+								"firstname": dsl.String("Child"),
+								"surname":   dsl.String("One"),
+								"dob":       dsl.Term("05/05/1980", `^\d{1,2}/\d{1,2}/\d{4}$`),
+							}}),
```

Comment:
I don't think it makes much difference but in the future probably worth using `EachLike` to be a bit more specific
```suggestion
							"children": dsl.EachLike(map[string]interface{}{
								"id":        dsl.Like(105),
								"uId":       dsl.Term("7000-0000-0002", `7\d{3}-\d{4}-\d{4}`),
								"firstname": dsl.String("Child"),
								"surname":   dsl.String("One"),
								"dob":       dsl.Term("05/05/1980", `^\d{1,2}/\d{1,2}/\d{4}$`),
							}, 1),
```